frontend/lib/norent/letter-builder/national-metadata.tsx

```diff
@@ -0,0 +1,83 @@
+import RawStateLawForBuilder from "../../../../common-data/norent-state-law-for-builder.json";
+import RawStateLawForLetter from "../../../../common-data/norent-state-law-for-letter.json";
+import RawStatePartnersForBuilder from "../../../../common-data/norent-state-partners-for-builder.json";
+import RawStateDocumentationRequirements from "../../../../common-data/norent-state-documentation-requirements.json";
+import RawStateLegalAidProviders from "../../../../common-data/norent-state-legal-aid-providers.json";
+import {
+  USStateChoice,
+  isUSStateChoice,
+} from "../../../../common-data/us-state-choices";
+
+type StateLawForBuilderEntry = {
+  linkToLegislation: string;
+  textOfLegislation: string;
+};
+
+type StateLawForLetterEntry = {
+  textOfLegislation1: string;
+  textOfLegislation2: string;
+  textOfLegislation3?: string;
+  textOfLegislation4?: string;
+  textOfLegislation5?: string;
+  textOfLegislation6?: string;
+};
+
+type StatePartnerForBuilderEntry = {
+  organizationName: string;
+  organizationWebsiteLink: string;
+};
+
+type StateDocumentationRequirementsEntry = {
+  doesTheTenantNeedToSendTheDocumentationToTheLandlord: boolean;
+  isDocumentationALegalRequirement: boolean;
+  numberOfDaysFromNonPaymentNoticeToProvideDocumentation?: number;
+};
+
+type StateLegalAidProviderEntry = {
+  localLegalAidProviderLink: string;
+};
+
+type StateMapping<T> = {
+  [k in USStateChoice]: T;
+};
+
+const StateLawForBuilder = RawStateLawForBuilder as StateMapping<
+  StateLawForBuilderEntry
+>;
+const StateLawForLetter = RawStateLawForLetter as StateMapping<
+  StateLawForLetterEntry
+>;
+const StatePartnersForBuilder = RawStatePartnersForBuilder as StateMapping<
+  StatePartnerForBuilderEntry
+>;
+const StateDocumentationRequirements = RawStateDocumentationRequirements as StateMapping<
+  StateDocumentationRequirementsEntry
+>;
+const StateLegalAidProviders = RawStateLegalAidProviders as StateMapping<
+  StateLegalAidProviderEntry
+>;
+
+/**
+ * Return the given string as a U.S. state choice, throwing an
+ * error if it's invalid.
+ */
+export const assertIsUSState = (state: string): USStateChoice => {
+  if (!isUSStateChoice(state)) {
+    throw new Error(`${state} is not a valid two-letter US state!`);
+  }
+  return state;
+};
+
+/**
+ * Return a big blob of metadata about NoRent.org-related information
+ * for the given U.S. state.
+ */
```

Comment:
Amazing! This looks super super helpful. Do you think it'll be an issue to call this function (and load the entire large blob) any time I need to grab data for a particular part of the flow? Is that going to slow things down in any way?

Reply:
Nope, it shouldn't--the way we're doing things in this initial version is just to send and parse all the metadata at page load time, so the downside is that the initial page load will be kinda larger and slower than ideal, but on the plus side you can use this function with reckless abandon!

We can revisit this strategy after the initial launch though :)